src/main/resources/schemas/nhinc/common/AuditQueryLog.xsd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="urn:gov:hhs:fha:nhinc:common:auditquerylog"
+            xmlns:tns="urn:gov:hhs:fha:nhinc:common:auditquerylog"
+            xmlns:adtmsg="http://nhinc.services.com/schema/auditmessage">
+    <xsd:import schemaLocation="../../../schemas/ihe/auditmessage.xsd" namespace="http://nhinc.services.com/schema/auditmessage"/>
+    <xsd:complexType name="QueryAuditEventsRequestType">
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventOutcomeIndicator" minOccurs="0" type="xsd:integer" />
+            <xsd:element name="EventBeginDate" type="xsd:date" />
+            <xsd:element name="EventEndDate" minOccurs="0" type="xsd:date" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsRequestType" type="tns:QueryAuditEventsRequestType"/>
+    
+    <xsd:complexType name="QueryAuditEventsSecureRequestType">
+        <xsd:sequence>
```

Comment:
QueryAuditEventsSecureRequestType complexType is not used in the xsd. Can we remove it?